javascript/1189-maximum-number-of-balloons.js

```diff
@@ -0,0 +1,25 @@
+// problem link https://leetcode.com/problems/maximum-number-of-balloons
+// time complexity O(n)
+
+var maxNumberOfBalloons = function(text) {
+
+    const  balloonCach = {};
+
+    for(let i = 0; i < text.length; i++) {
+        if('balloon'.includes(text[i])) {
```

Comment:
I edited the file and replaced the string with the set so now it takes O(1) time.  But, since we are having a constant string such as a "balloon" isn't the time complexity will be O(1) since our input is not increasing, I mean it's a constant "balloon".

Reply:
It will be O(1) if your string is 'ballon', but it is better to assume you are working with a string of O(N)